docs/UserGuide.md

```diff
@@ -528,8 +532,13 @@ Lists all incomes with the category `prize` and/or `gift`, with amounts between
 
 ### 4.5 Bookmark Expense
 
-Fine<span>$</span><span>$</span>e's Bookmark Expense feature is used to store frequent expenses that the user makes such as paying of monthly phone bills or buying bubble tea weekly.
-The user will be then be able to edit, delete and convert a bookmarked expense to conveniently add it into Fine<span>$</span><span>$</span>e's expense list.
+Fine<span>$</span><span>$</span>e's Bookmark Expense feature are templates to create expenses that the user frequently makes such as paying of monthly phone bills or buying bubble tea weekly.
+The user will be then be able to edit, delete and convert a bookmark expense to conveniently add it into Fine<span>$</span><span>$</span>e's expense list.
+
+Fine\$\$e records the following information about each of your bookmark expenses:
+1. **Title**: A descriptive title, to remind you what the bookmark expense was about.
```

Comment:
```suggestion
1. **Title**: A descriptive title, to remind you what the bookmark expense is about.
```
I suggest changing the descriptions in this list to present tense since, unlike normal expenses, bookmark expenses are recurring hence can occur in the present.

Do mirror this change for bookmark incomes as well.